test/integration/targets/ansible-playbook-callbacks/runme.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -eux
+
+export ANSIBLE_CALLBACK_PLUGINS=../ansible/callback_plugins
```

Comment:
@mattclay Should `callback_debug` be moved to a common path for all targets to use?

Reply:
It wouldn't hurt to move that callback plugin into its own hidden test target, so it could be re-used without adding the dependency on the `ansible` target.